### DISCUSSION TITLE
Staying in touch outside of Facebook?

### DISCUSSION THREAD

**Discussion by wohali** [keirf/greaseweazle#63] [Ideas]
Hey Keir, nice to make your acquaintance. I've been doing builds of GW F7 (v1.1) boards and intend to start doing my own designs in the future. I'd love to help with HW and SW improvement here and over at Disk-Utilities.

Is there any chance of staying in touch outside of that platform, or possibly building a non-FB community? I (and many that I know, including some extremely knowledgeable people) refuse to use Facebook. (One of them even described the entire GW effort as a "non-starter" because of this part of your community.)

I've noticed that activity on GitHub is very limited, and that is very disappointing. GitHub has a new Discussions function that might be very useful to this project -- see [here](https://docs.github.com/en/free-pro-team@latest/discussions/quickstart) for more details.

**Comment by qbarnes** [comment]
Yes, this is my problem as well with having active participation in this community.  I have never had a Facebook account, and never will, due to their unsavory business practices.

**Comment by keirf** [comment]
There we are. A Discussion :) This seems a good idea; I wasn't aware of this GH feature.

**Reply by wohali** [reply]
It's brand new! We've been beta testing it over in Apache CouchDB for months, and it's finally ready for prime time.

Thanks for enabling this -- looking forward to staying in touch!
